docs/source/index.rst

```diff
@@ -158,6 +158,14 @@ Or directly from conda
 
    image/*
 
+.. toctree::
```

Comment:
We've sorted this section in an alphabetical order in #1298, so maybe we can keep it here?